src/libclang/class_parser.cpp

```diff
@@ -119,6 +119,7 @@ std::unique_ptr<cpp_entity> detail::parse_cpp_class(const detail::parse_context&
     auto is_templated = (clang_getTemplateCursorKind(cur) != CXCursor_NoDeclFound
                          || !clang_Cursor_isNull(clang_getSpecializedCursorTemplate(cur)));
     auto is_friend    = clang_getCursorKind(parent_cur) == CXCursor_FriendDecl;
+	auto is_abstract  = clang_CXXRecord_isAbstract(cur);
```

Comment:
Is this available with all libclang versions or was it recently introduced? If it is new, please check the libclang version and #ifdef it away if necessary.